crates/iota-sdk-ffi/src/types/graphql.rs

```diff
@@ -711,39 +668,54 @@ impl From<GQLAddress> for iota_graphql_client::query_types::GQLAddress {
     }
 }
 
-#[derive(Clone, Debug, uniffi::Record)]
+#[uniffi::remote(Record)]
 pub struct MoveObject {
     #[uniffi(default = None)]
-    pub bcs: Option<String>,
-}
-
-impl From<iota_graphql_client::query_types::MoveObject> for MoveObject {
-    fn from(value: iota_graphql_client::query_types::MoveObject) -> Self {
-        MoveObject {
-            bcs: value.bcs.map(|v| v.0),
-        }
-    }
-}
-
-impl From<MoveObject> for iota_graphql_client::query_types::MoveObject {
-    fn from(value: MoveObject) -> Self {
-        iota_graphql_client::query_types::MoveObject {
-            bcs: value.bcs.map(iota_graphql_client::query_types::Base64),
-        }
-    }
+    pub bcs: Option<Base64>,
 }
 
 #[derive(Clone, Debug, derive_more::From, uniffi::Object)]
 pub struct ProtocolConfigs(pub iota_graphql_client::query_types::ProtocolConfigs);
 
-#[derive(Clone, Debug, derive_more::From, uniffi::Object)]
-pub struct CoinMetadata(pub iota_graphql_client::query_types::CoinMetadata);
+#[uniffi::remote(Record)]
+pub struct CoinMetadata {
+    pub decimals: Option<i32>,
+    pub description: Option<String>,
+    pub icon_url: Option<String>,
+    pub name: Option<String>,
+    pub symbol: Option<String>,
+    pub supply: Option<BigInt>,
```

Comment:
Add `#[uniffi(default = None)]` for them?

Reply:
One might want to create it manually for some known cases to not fetch it every time again or to perhaps for some tests and then it would be more handy with default = None